extensions/parallels-virtual-machines/package.json

```diff
@@ -5,9 +5,21 @@
   "description": "Launch and control virtual machines in Parallels Desktop.",
   "icon": "command-icon.png",
   "author": "danpalmer",
-  "categories": [
-    "Productivity",
-    "Developer Tools"
+  "contributors": ["jakucermak"],
+  "categories": ["Productivity", "Developer Tools"],
+  "preferences": [
+    {
+      "name": "IconColorType",
+      "required": false,
+      "title": "Icon Color Type",
+      "description": "Which colors are used for OS icons Mono/Colored",
+      "type": "dropdown",
+      "data": [
+        { "title": "Colored", "value": "0" },
+        { "title": "Mono", "value": "1" }
```

Comment:
Could we use more descriptive values rather than "0" and "1" – being numbers is a bit confusing too. Maybe "mono" and "colour"/"color"?